classes/entity/enemy/bobomb/bobomb.gd

```diff
@@ -6,15 +6,30 @@ extends EntityEnemyWalk
 # When their fuse is lit, they do not lose focus on the player.
 # They can be struck to send them flying a short distance before exploding.
 
+# When two bombs play the buildup sound at once, the sound gets confusing.
+# Would be nice to have it choose from two or three different pitches so
+# the player can differentiate which sound belongs to which bomb, but
```

Comment:
Perhaps having a bob-omb flash red in time with the sounds might help? That wouldn't solve this directly, but would communicate the same information.

Reply:
Good thought; definitely for a separate PR though 😛